.github/actions/check_docstrings/action.yml

```diff
@@ -0,0 +1,63 @@
+name: Check Docstrings
+inputs:
+    python-version:
+        description: 'The version of Python to use for the workflow.'
+        default: '3.10'
+        required: false
+        type: string
+    repository:
+        description: 'The repository to check the docstrings for.'
+        required: True
+        type: string
+    branch:
+        description: 'The branch to check the docstrings for.'
+        required: false
+        type: string
+        default: ''
```

Comment:
What happens if `''` is passed?

Reply:
branch is passed in as a `ref` to actions/checkout.  So, if nothing is passed in, it defaults to the `default` branch